spec/unit/recipes/default_spec.rb

```diff
@@ -5,18 +5,19 @@
 # Copyright (c) 2015 Irving Popovetsky, All Rights Reserved.
 
 require 'spec_helper'
+require 'mixlib/versioning' # new chef-ingredient makes heavy use of this library
```

Comment:
@schisamo @sersut Perhaps it's worth considering leveraging `mixlib-versioning` for version-related matters in Chef, in general, and including it in Chef/ChefDK omnibus packages?